package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "@nuxt/content": "^1.14.0",
     "@nuxtjs/markdownit": "^2.0.0",
+    "axios": "^0.21.1",
```

Comment:
instead of using `axios`, use the `fetch` api (example of use with the async fetch on the current main branch `pages/index.vue`)